fboss/agent/platforms/sai/SaiBcmIcecube800bcPlatformPort.h

```diff
@@ -20,6 +20,16 @@ class SaiBcmIcecube800bcPlatformPort : public SaiBcmPlatformPort {
   void linkStatusChanged(bool up, bool adminUp) override;
   void externalState(PortLedExternalState lfs) override;
 
+  uint32_t getPhysicalLaneId(uint32_t chipId, uint32_t logicalLane)
+      const override {
+    if (getPortType() == cfg::PortType::MANAGEMENT_PORT) {
+      // The management port core ID is 64
+      // However, the correct lane numbering in TH6 start from 514(514-517)
+      return SaiBcmPlatformPort::getPhysicalLaneId(chipId, logicalLane) + 1;
```

Comment:
@Rancho333-CLS is this a temporary workaround or a permanent fix?

Reply:
This is a permanent fix. In FBOSS, the hardware lane list of a port is calculated based on the core ID. On TH6, the lane IDs of the management ports are not sequential, so this function is overridden. Other TH6 projects also need to do the same.